lib/licensee/projects/git_project.rb

```diff
@@ -68,7 +68,16 @@ def load_file(file)
       #  :name - the file's path relative to the repo root
       #  :oid  - the file's OID
```

Comment:
I should document here and elsewhere that we're also including `:dir` in the hash.

Reply:
Done in the latest version.

Reply:
This looks like a potentially significant API change. Is it necessary?

Reply:
Which change are you referring to? If it is that we're including `:dir` in the hash, as far as I could tell all 3 backends already included `:dir`, it just wasn't documented.

If it's that I changed `files` to also include the `LICENSES/` files, yes this does change things. Would you prefer I could instead add a new API to fetch files from a subdirectory?